LiskHQ/lisk-hub
# Implement the my account component - closes #2038

### What issue have I solved?

-- #2038 

### How have I implemented/fixed it?

PR includes the implementation of the new dashboard component called My Account Details, this component is just informative and will display the enable currencies that the user has, right now is based on Lisk and BTC (what can be enable/disabled) and this will show the crypto currency and the actual balance.

If the user has disabled BTC then this will not show up in the component and only LSK.

If the user logout then this component won't be visible, in other words this component only will work/show up after Sign in

### How has this been tested?

1. Sign In into the app.
2. Go to dashboard and at the end of the URL add the V2 like this -> .../#/dashboardV2. Then you will see the new component.
3. For enable or disable BTC, you can dispatch this function in the browser redux tool.
For enable BTC
```
{ type: 'SETTINGS_UPDATED', data: { token: { list: { BTC: true } } } }
```
For disable BTC
```
{ type: 'SETTINGS_UPDATED', data: { token: { list: { BTC: false } } } }
```
Then you will see the change in the component.

**NOTE:** Is important to have the BTC flag in the local storage set it in true (btc: true) if not you wont be able to see BTC information
### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
